Lib/tkinter/__init__.py

```diff
@@ -185,8 +185,7 @@ class EventType(str, enum.Enum):
     Deactivate = '37',
     MouseWheel = '38',
 
-    def __str__(self):
-        return self.name
+    __str__ = str.__str__
```

Comment:
This line is unnecessary if subclassing the new `StrEnum`, which my patch did.  Just remove the `def __str__` and we're good.